eng/pipelines/templates/steps/test-proxy-local-tool.yml

```diff
@@ -26,14 +26,14 @@ steps:
 
     - pwsh: |
         Start-Process $(Build.BinariesDirectory)/test-proxy/test-proxy.exe `
-          -ArgumentList "--storage-location ${{ parameters.rootFolder }}" `
+          -ArgumentList "--storage-location ${{ parameters.rootFolder }} -U" `
           -NoNewWindow -PassThru -RedirectStandardOutput $(Build.SourcesDirectory)/test-proxy.log
       displayName: 'Run the testproxy - windows'
       condition: and(succeeded(), eq(variables['Agent.OS'],'Windows_NT'))
 
     # nohup does NOT continue beyond the current session if you use it within powershell
     - bash: |
-        nohup $(Build.BinariesDirectory)/test-proxy/test-proxy > $(Build.SourcesDirectory)/test-proxy.log &
+        nohup $(Build.BinariesDirectory)/test-proxy/test-proxy &>$(Build.SourcesDirectory)/test-proxy.log &
```

Comment:
Why is the & at the end of this line? Perhaps it some bash think I don't understand but I'm just curious.

Reply:
It's been a bit, but it's to make `nohup` run in background 👍